clusterman/draining/queue.py

```diff
@@ -251,7 +249,6 @@ def process_termination_queue(
                 logger.info(f'Kubernetes hosts to delete k8s node and terminate: {host_to_terminate}')
                 try:
```

Comment:
you could probably just remove this try catch entirely

Reply:
Fixed